File: src/b1happynumber/Java/Main.java
public class Main {

    public static void main(String[] args) {
        int[] inputs = { 1, 5, 19, 25, 7 };

        for (int input : inputs) {
            System.out.println("Input Number: " + input);
            System.out.println("Is it a happy number? " + HappyNumberHelper.isHappyNumber(input));
            System.out.println("-".repeat(50));
        }
    }
}

File: src/b1happynumber/TS/main.ts
import { isHappyNumber } from './happyNumberHelper';

function main(): void {
    const inputs: number[] = [1, 5, 19, 25, 7];
    inputs.forEach((input, i) => {
        console.log(`Input Number: ${input}`);
        console.log(`Is it a happy number? ${isHappyNumber(input)}`);
        console.log("-".repeat(50));
    });
}

main();
